src/expression.d

```diff
@@ -7602,12 +7615,14 @@ public:
 
     override Expression semantic(Scope* sc)
     {
-        const(char)* name;
-        StringExp se;
         static if (LOGSEMANTIC)
         {
             printf("FileExp::semantic('%s')\n", toChars());
         }
+        const(char)* name;
```

Comment:
urgh again with the pascal convention of putting all variables at the beginning...